scripts/us_census/acs5yr/subject_tables/s2201/README.md

```diff
@@ -0,0 +1,28 @@
+# Importing Census ACS5Year Table S2201
+
+## Input Data
+
+The data is downloaded from
+`https://www.census.gov/acs/www/data/data-tables-and-tools/subject-tables/` and
+is stored in GCS at
+`datcom-csv/census/acs5yr/subject_tables/s2201/`.
+Currently, we have data from 2010-2019 and observations for the Statistical
+Variables listed in
+`stat_vars.csv`.
```

Comment:
What geos do these cover?  Can we summarize the format of the downloaded CSVs?

Reply:
updated description